cognite_toolkit/_cdf_tk/storageio/_asset_centric.py

```diff
@@ -110,7 +110,7 @@ def download_iterable(self, selector: AssetCentricSelector, limit: int | None =
     def upload_items(self, data_chunk: AssetWriteList, selector: AssetCentricSelector) -> None:
         if not data_chunk:
             return
-        self.client.assets.create(data_chunk)
+        self.client.assets.upsert(data_chunk, mode="patch")
```

Comment:
Does this constitute a change in behavior that could have side effects elsewhere?

Reply:
Sorry I am a bit confused. You are linking to this test on a different git-hash. That test is updated to comply as shown below